pkg/collector/corechecks/net/ntp/ntp_local_defined_server_windows_test.go

```diff
@@ -13,25 +13,95 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestGetLocalDefinedNTPServers(t *testing.T) {
-	servers, err := getLocalDefinedNTPServers()
-	assert.NoError(t, err)
-	assert.NotEmpty(t, servers)
+// Test the helper function that parses registry values
+func TestGetNptServersFromRegKeyValue(t *testing.T) {
+	testCases := []struct {
+		name            string
+		regKeyValue     string
+		expectedServers []string
+		expectError     bool
+	}{
+		{
+			name:            "Single server with flags",
+			regKeyValue:     "time.windows.com,0x9",
+			expectedServers: []string{"time.windows.com"},
+			expectError:     false,
+		},
+		{
+			name:            "Multiple servers space-separated",
+			regKeyValue:     "pool.ntp.org time.windows.com time.apple.com time.google.com",
+			expectedServers: []string{"pool.ntp.org", "time.windows.com", "time.apple.com", "time.google.com"},
+			expectError:     false,
+		},
+		{
+			name:            "Multiple servers with flags",
+			regKeyValue:     "ntp1.example.com,0x1 ntp2.example.com,0x9",
+			expectedServers: []string{"ntp1.example.com", "ntp2.example.com"},
+			expectError:     false,
+		},
+		{
+			name:            "Empty string",
+			regKeyValue:     "",
+			expectedServers: nil,
+			expectError:     true,
+		},
+		{
+			name:            "Whitespace only",
+			regKeyValue:     "   \t\n   ",
+			expectedServers: nil,
+			expectError:     true,
+		},
+		{
+			name:            "Server with port",
+			regKeyValue:     "ntp.example.com:123,0x9",
```

Comment:
do we support a custom port in the host field? It looks like the check has a separate option for it.